inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -52,6 +58,44 @@ bool compare_rt_keys(const std::shared_ptr<ngraph::Node>& node1, const std::shar
     return true;
 }
 
+bool less_by_name(
+    const std::shared_ptr<ngraph::op::v0::Result>& l,
+    const std::shared_ptr<ngraph::op::v0::Result>& r) {
+    return l->get_friendly_name() < r->get_friendly_name();
+}
+
+template <typename T>
+std::string to_str(const T& v) {
+    return std::to_string(v);
+}
+
+std::pair<bool, std::string> error(std::string s) {
+    return {false, std::move(s)};
+}
+
+std::string typeInfoToStr(const ngraph::Node::type_info_t& typeInfo) {
+    return std::string(typeInfo.name) + "/" + to_str(typeInfo.version);
+}
+
+template <typename Node>
+std::string name(const Node& n) {
```

Comment:
Why do we need it? I think function that call another functions are useless especially in this case.

Reply:
From my perspective this function reduce boilerplate code.